libs/explore/feature/src/lib/explore.component.ts

```diff
@@ -181,19 +186,55 @@ export class ExploreComponent implements OnInit {
       });
   
       this.AuPairArray = this.AuPairArray.filter((element) => {
-        return element.payRate > this.minPayrate && element.payRate < this.maxPayrate;
+        return element.payRate >= this.minPayrate && element.payRate <= this.maxPayrate;
       });
     }
     
     this.closeMenu();
   }
 
+  ageFilter(formData : any)
+  {
+    this.AuPairArray.splice(0);
+    this.restoredAuPairArray.forEach(val => this.AuPairArray.push(Object.assign({}, val)));
+
+    if(formData.max_age === undefined || formData.max_age === '')
+    {
+      this.maxAge = 18;
```

Comment:
I don't quite understand what maxAge does, does this filter mean the max age for an au pair is 18?

Reply:
This is if someone applies the max age filter without moving the slider, it will set max age to 18, which is where the slider starts at